discovery-provider/plugins/network-monitoring/src/utils.ts

```diff
@@ -193,6 +193,8 @@ export const getEnv = () => {
         password: process.env['FDB_PASSWORD'] || '',
     }
 
+    const foundationNodesEnv = process.env['FOUNDATION_NODES'] || ''
```

Comment:
should we make this envvar name more clear as `FOUNDATION_NODE_SP_IDS` and same for variable names

Reply:
Agreed